pkg/templates/images/images.go

```diff
@@ -377,26 +373,26 @@ func optionalResources() map[Resource]map[string]string {
 		WeaveNetCNINPC:  {"*": "docker.io/weaveworks/weave-npc:2.8.1"},
 
 		// Calico VXLAN
-		CalicoVXLANCNI:        {"*": "quay.io/calico/cni:v3.28.2"},
-		CalicoVXLANController: {"*": "quay.io/calico/kube-controllers:v3.28.2"},
-		CalicoVXLANNode:       {"*": "quay.io/calico/node:v3.28.2"},
+		CalicoVXLANCNI:        {"*": "quay.io/calico/cni:v3.29.2"},
+		CalicoVXLANController: {"*": "quay.io/calico/kube-controllers:v3.29.2"},
+		CalicoVXLANNode:       {"*": "quay.io/calico/node:v3.29.2"},
```

Comment:
Do we really need this now that the addon is removed?

Reply:
Oh yeah, thanks for pointing out. Removed!